.codeclimate.yml

```diff
@@ -10,6 +10,9 @@ engines:
         enabled: false
   radon:
     enabled: true
+    config:
+      python_version: 3
+      threshold: "D"
```

Comment:
This is just the lowest "grade" code climate can give us?

Reply:
Good question. I intended to share this info.

Default is "B". D isn't the lowest, F is the lowest grade.
http://radon.readthedocs.io/en/latest/commandline.html#the-cc-command